logger/logger.go

```diff
@@ -47,6 +52,55 @@ func onConfigUpdated(cfg Config) {
 	}
 }
 
+type GcpFormatter struct {
+}
+
+type GcpEntry struct {
+	Data      map[string]interface{} `json:"data,omitempty"`
+	Message   string                 `json:"message,omitempty"`
+	Severity  GcpSeverity            `json:"severity,omitempty"`
+	Timestamp string                 `json:"timestamp,omitempty"`
+	// CHECK: Context
```

Comment:
It is not clear what is happening here. Is this a TODO that would be addressed in an followup commit or in a follow PR?